### PR TITLE
Ci for rhtap build pipeline

### PR DESCRIPTION
**Pull request by prietyc123**

### CODE REVIEW

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-shellcheck

File: core-services/prow/02_config/janus-idp/_prowconfig.yaml [hunk 1]
```diff
@@ -14,3 +14,15 @@ tide:
     - needs-rebase
     orgs:
     - janus-idp
+  - labels:
```
Comment (rhopp) [inline core-services/prow/02_config/janus-idp/_prowconfig.yaml:17]: Is this change intentional?

Comment (prietyc123) [reply core-services/prow/02_config/janus-idp/_prowconfig.yaml]: Reverted with original change. Thanks for pointing it out

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-yamllint

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-yamllint

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-shellcheck

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-validate-rhtap-sample-pipelines

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-validate-rhtap-sample-pipelines

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-validate-rhtap-sample-pipelines

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-validate-rhtap-sample-pipelines

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-validate-rhtap-sample-pipelines

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-validate-rhtap-sample-pipelines

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-validate-rhtap-sample-pipelines

**Comment by prietyc123** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-validate-rhtap-sample-pipelines

**Comment by prietyc123** [comment]
This PR is blocked on https://github.com/redhat-appstudio/tssc-sample-pipelines/pull/23

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-shellcheck

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-yamllint

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-shellcheck

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-yamllint

**Review by xinredhat — APPROVED** [review]

**Comment by flacatus** [comment]
/lgtm
aapprove
/pj-rehearse ack
